guides/common/modules/proc_deploying-the-ca-certificate-on-a-host-manually.adoc

```diff
@@ -0,0 +1,63 @@
+[id="deploying-the-ca-certificate-on-a-host-manually"]
+= Deploying the CA certificate on a host manually
+
+You can deploy the CA certificate on the host manually by rendering a public provisioning template, which will provide the certificate to your host.
+
+.Prerequisites
+* You have root access on your host.
+* The CA certificate has been changed on {ProjectServer}.
+For more information, see {AdministeringDocURL}planning-for-ca-certificate-renewal[Planning for CA certificate renewal] in _{AdministeringDocTitle}_.
+
+.Procedure
+. Log in to you host by using SSH.
+. Download the certificate on your host:
```

Comment:
Similar as above, perhaps we should say get the certificate on your host using insecure HTTP, SCP or some other secure method?

Reply:
This is the standard way we provide guidelines for manual config on hosts. I'm sure that advanced users would be able to adapt it as they need.

Reply:
In the next line, we are showing them how to download the CA certificate. Which is going to be inherently insecure, and users may not realize that because they trust our documentation. I think this is an area where we should be more clear, and use the right language in order to express that **care** needs to be taken.

Reply:
If they plan the replacement of their custom CA cert properly as described in the _Planning_ section and mentioned in Prerequisites, what reasons are there left for the download to be insecure?

Reply:
Under those conditions yes - but this snippet is not specific to a user having done that. This is a generic "how to deploy a CA on a host manually" section based on how it's titled. Thus, I feel we shouldn't assume anything and give users all the info and tools to handle all of the possibilities.

Reply:
Actually, yes, this one doesn't assume the Planning. I mixed it up. On it!

Reply:
@ehelms So how exactly should the user get the CA certificate to the host securely?

Reply:
Assuming the endpoint isn't secure, then the user should just copy (scp) the cert from ProjectServer's `/var/html/www/pub` to the host instead of using the endpoint at all?
cc @ShimShtein

Reply:
Done. Please, review.